MANIFEST.in

```diff
@@ -0,0 +1,7 @@
+include README.md COPYING
+global-include CMakeLists.txt *.cmake
+recursive-include external/catch2 *
+recursive-include lib *
```

Comment:
Is this file in use at all? It does not seem to be referenced in the PR at all?

Reply:
yeah... https://docs.python.org/2/distutils/sourcedist.html.. "but usually you will want to specify additional files to distribute. The typical way to do this is to write a manifest template, called MANIFEST.in by default. ...The sdist command processes this template and generates a manifest based on its instructions and what it finds in the filesystem."